src/FirebaseArray.js

```diff
@@ -7,24 +7,25 @@
    *
    * Internally, the $firebase object depends on this class to provide 5 $$ methods, which it invokes
    * to notify the array whenever a change has been made at the server:
-   *    $$added - called whenever a child_added event occurs
-   *    $$updated - called whenever a child_changed event occurs
-   *    $$moved - called whenever a child_moved event occurs
-   *    $$removed - called whenever a child_removed event occurs
+   *    $$added - called whenever a child_added event occurs, returns the new record, or null to cancel
```

Comment:
i guess it's unclear, but the point is that returning null will cancel the add operation (nothing gets added to the array and $$process is not invoked)